src/app/mainPage/rankingCard/RankingCard.tsx

```diff
@@ -0,0 +1,35 @@
+import { LinearProgress, Typography } from '@mui/material';
+import EventsIcon from '@mui/icons-material/EmojiEvents';
+
+import { styles, Wrapper } from './RankingCard.styles';
+import { RankingCardProps } from './RankingCard.types';
+import { useLocale } from 'hooks';
+import { Goal } from 'api/ranking/ranking.types';
+import { Container } from 'ui';
+
+export const RankingCard = ({ name, points, rank }: RankingCardProps) => {
+  const { formatMessage } = useLocale();
+
+  return (
+    <Container>
+      <Wrapper>
```

Comment:
Przyjęło się, że jeśli korzystamy z MUI, to nie korzystamy z divów, tylko zawsze z `Box` i style pod `sx`. Chodzi o to, żeby mieć wspólny base. To pozwala na proste modyfikacje, jeśli musiałbyś np zmodyfikować wszystkie divy a apce